### ISSUE TITLE
CRIO Time_Format incorrect

### ISSUE THREAD

**Issue by frankreno** [SumoLogic/sumologic-kubernetes-collection#1417]
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/10101015924ebe6c92bcf9ce9a757d5da82ba302/deploy/helm/sumologic/values.yaml#L938

Correct Time_Format should be `%Y-%m-%dT%H:%M:%S.%LZ`. Otherwise parsing fails.

**Comment by kkujawa-sumo** [comment]
`Time_Format` cannot be in quotes.
`Time_Format` for CRI-O: `%Y-%m-%dT%H:%M:%S.%L%z`
`Time_Format` for containerd: `%Y-%m-%dT%H:%M:%S.%LZ`

**CRI-O** logs e.g.
```
2021-02-16T08:59:44.287589429+00:00 stdout F example: 24 Tue Feb 16 08:59:44 UTC 2021
2021-02-16T08:59:45.290025982+00:00 stdout F example: 25 Tue Feb 16 08:59:45 UTC 2021
2021-02-16T08:59:46.291424975+00:00 stdout F example: 26 Tue Feb 16 08:59:46 UTC 2021
2021-02-16T08:59:47.292686634+00:00 stdout F example: 27 Tue Feb 16 08:59:47 UTC 2021
2021-02-16T08:59:48.294373602+00:00 stdout F example: 28 Tue Feb 16 08:59:48 UTC 2021
```
correct Parser for **CRI-O l**ogs:
```
 [PARSER]
          Name         crio
          Format       regex
          Regex        ^(?<time>[^ ]+) (?<stream>stdout|stderr) (?<logtag>[^ ]*) (?<log>.*)$
          Time_Key     time
          Time_Format  %Y-%m-%dT%H:%M:%S.%L%z
```

stdout from Fluent Bit:

```
[0] containers.var.log.containers.example-kk_default_count-1250ef234c6429d0022d660b06caab56a790097214ea683c37c1b4ee84735f8d.log: [1613564349.605459824, {"stream"=>"stdout", "logtag"=>"F", "log"=>"example-kk12: 16047 Wed Feb 17 12:19:09 UTC 2021"}]
[0] containers.var.log.containers.example-kk_default_count-1250ef234c6429d0022d660b06caab56a790097214ea683c37c1b4ee84735f8d.log: [1613564350.606409986, {"stream"=>"stdout", "logtag"=>"F", "log"=>"example-kk12: 16048 Wed Feb 17 12:19:10 UTC 2021"}]
[0] containers.var.log.containers.example-kk_default_count-1250ef234c6429d0022d660b06caab56a790097214ea683c37c1b4ee84735f8d.log: [1613564351.608882243, {"stream"=>"stdout", "logtag"=>"F", "log"=>"example-kk12: 16049 Wed Feb 17 12:19:11 UTC 2021"}]
[0] containers.var.log.containers.example-kk_default_count-1250ef234c6429d0022d660b06caab56a790097214ea683c37c1b4ee84735f8d.log: [1613564352.610323599, {"stream"=>"stdout", "logtag"=>"F", "log"=>"example-kk12: 16050 Wed Feb 17 12:19:12 UTC 2021"}]
[0] containers.var.log.containers.example-kk_default_count-1250ef234c6429d0022d660b06caab56a790097214ea683c37c1b4ee84735f8d.log: [1613564353.612071381, {"stream"=>"stdout", "logtag"=>"F", "log"=>"example-kk12: 16051 Wed Feb 17 12:19:13 UTC 2021"}]
```

wrong Parser for **CRI-O** logs:
```
[PARSER]
          Name         crio
          Format       regex
          Regex        ^(?<time>[^ ]+) (?<stream>stdout|stderr) (?<logtag>[^ ]*) (?<log>.*)$
          Time_Key     time
          Time_Format  "%Y-%m-%dT%H:%M:%S.%L%z"
```

stdout from Fluent Bit:
```
[2021/02/17 12:22:38] [error] [parser] cannot parse '2021-02-17T12:22:38.654007733+00:00'
[2021/02/17 12:22:38] [ warn] [parser:crio] invalid time format "%Y-%m-%dT%H:%M:%S.%L%z" for '2021-02-17T12:22:38.654007733+00:00'
[0] containers.var.log.containers.example-kk_default_count-1250ef234c6429d0022d660b06caab56a790097214ea683c37c1b4ee84735f8d.log: [1613564558.654445636, {"stream"=>"stdout", "logtag"=>"F", "log"=>"example-kk12: 16247 Wed Feb 17 12:22:38 UTC 2021"}]
[2021/02/17 12:22:39] [error] [parser] cannot parse '2021-02-17T12:22:39.655979891+00:00'
[2021/02/17 12:22:39] [ warn] [parser:crio] invalid time format "%Y-%m-%dT%H:%M:%S.%L%z" for '2021-02-17T12:22:39.655979891+00:00'
[0] containers.var.log.containers.example-kk_default_count-1250ef234c6429d0022d660b06caab56a790097214ea683c37c1b4ee84735f8d.log: [1613564559.656108733, {"stream"=>"stdout", "logtag"=>"F", "log"=>"example-kk12: 16248 Wed Feb 17 12:22:39 UTC 2021"}]
[2021/02/17 12:22:40] [error] [parser] cannot parse '2021-02-17T12:22:40.659469615+00:00'
[2021/02/17 12:22:40] [ warn] [parser:crio] invalid time format "%Y-%m-%dT%H:%M:%S.%L%z" for '2021-02-17T12:22:40.659469615+00:00'
[0] containers.var.log.containers.example-kk_default_count-1250ef234c6429d0022d660b06caab56a790097214ea683c37c1b4ee84735f8d.log: [1613564560.659618376, {"stream"=>"stdout", "logtag"=>"F", "log"=>"example-kk12: 16249 Wed Feb 17 12:22:40 UTC 2021"}]
[2021/02/17 12:22:41] [error] [parser] cannot parse '2021-02-17T12:22:41.662160903+00:00'
[2021/02/17 12:22:41] [ warn] [parser:crio] invalid time format "%Y-%m-%dT%H:%M:%S.%L%z" for '2021-02-17T12:22:41.662160903+00:00'
```

**containerd** logs e.g.

```
2021-02-16T09:21:16.519721603Z stdout F example: 11 Tue Feb 16 09:21:16 UTC 2021
2021-02-16T09:21:17.521307236Z stdout F example: 12 Tue Feb 16 09:21:17 UTC 2021
2021-02-16T09:21:18.522445945Z stdout F example: 13 Tue Feb 16 09:21:18 UTC 2021
2021-02-16T09:21:19.523759881Z stdout F example: 14 Tue Feb 16 09:21:19 UTC 2021
2021-02-16T09:21:20.545525544Z stdout F example: 15 Tue Feb 16 09:21:20 UTC 2021
2021-02-16T09:21:21.546984796Z stdout F example: 16 Tue Feb 16 09:21:21 UTC 2021
2021-02-16T09:21:22.549417066Z stdout F example: 17 Tue Feb 16 09:21:22 UTC 2021
2021-02-16T09:21:23.550631431Z stdout F example: 18 Tue Feb 16 09:21:23 UTC 2021
```

correct Parser for **containerd** logs:
```
[PARSER]
          Name         containerd
          Format       regex
          Regex        ^(?<time>[^ ]+) (?<stream>stdout|stderr) (?<logtag>[^ ]*) (?<log>.*)$
          Time_Key     time
          Time_Format  %Y-%m-%dT%H:%M:%S.%LZ
```

stdout from Fluent Bit:
```
[0] containers.var.log.containers.example-kk_default_count-ab9e855d67b387ca0d2cf624e7b90a0f593ff7fd0a0a5ab0c50afce12eb9524b.log: [1613563937.497587104, {"stream"=>"stdout", "logtag"=>"F", "log"=>"example-kk12: 4906 Wed Feb 17 12:12:17 UTC 2021"}]
[0] containers.var.log.containers.example-kk_default_count-ab9e855d67b387ca0d2cf624e7b90a0f593ff7fd0a0a5ab0c50afce12eb9524b.log: [1613563938.501247483, {"stream"=>"stdout", "logtag"=>"F", "log"=>"example-kk12: 4907 Wed Feb 17 12:12:18 UTC 2021"}]
[0] containers.var.log.containers.example-kk_default_count-ab9e855d67b387ca0d2cf624e7b90a0f593ff7fd0a0a5ab0c50afce12eb9524b.log: [1613563939.502288817, {"stream"=>"stdout", "logtag"=>"F", "log"=>"example-kk12: 4908 Wed Feb 17 12:12:19 UTC 2021"}]
[0] containers.var.log.containers.example-kk_default_count-ab9e855d67b387ca0d2cf624e7b90a0f593ff7fd0a0a5ab0c50afce12eb9524b.log: [1613563940.504221286, {"stream"=>"stdout", "logtag"=>"F", "log"=>"example-kk12: 4909 Wed Feb 17 12:12:20 UTC 2021"}]
[0] containers.var.log.containers.example-kk_default_count-ab9e855d67b387ca0d2cf624e7b90a0f593ff7fd0a0a5ab0c50afce12eb9524b.log: [1613563941.504998042, {"stream"=>"stdout", "logtag"=>"F", "log"=>"example-kk12: 4910 Wed Feb 17 12:12:21 UTC 2021"}]
```

wrong Parser for **containerd** logs:
```
[PARSER]
          Name         containerd
          Format       regex
          Regex        ^(?<time>[^ ]+) (?<stream>stdout|stderr) (?<logtag>[^ ]*) (?<log>.*)$
          Time_Key     time
          Time_Format  "%Y-%m-%dT%H:%M:%S.%LZ"
```

stdout from Fluent Bit:
```
[2021/02/17 12:14:55] [ warn] [parser:containerd] invalid time format "%Y-%m-%dT%H:%M:%S.%LZ" for '2021-02-17T12:14:55.834603863Z'
[0] containers.var.log.containers.example-kk_default_count-ab9e855d67b387ca0d2cf624e7b90a0f593ff7fd0a0a5ab0c50afce12eb9524b.log: [1613564095.834708561, {"stream"=>"stdout", "logtag"=>"F", "log"=>"example-kk12: 5064 Wed Feb 17 12:14:55 UTC 2021"}]
[2021/02/17 12:14:56] [error] [parser] cannot parse '2021-02-17T12:14:56.835433467Z'
[2021/02/17 12:14:56] [ warn] [parser:containerd] invalid time format "%Y-%m-%dT%H:%M:%S.%LZ" for '2021-02-17T12:14:56.835433467Z'
[0] containers.var.log.containers.example-kk_default_count-ab9e855d67b387ca0d2cf624e7b90a0f593ff7fd0a0a5ab0c50afce12eb9524b.log: [1613564096.835556678, {"stream"=>"stdout", "logtag"=>"F", "log"=>"example-kk12: 5065 Wed Feb 17 12:14:56 UTC 2021"}]
[2021/02/17 12:14:57] [error] [parser] cannot parse '2021-02-17T12:14:57.836555741Z'
[2021/02/17 12:14:57] [ warn] [parser:containerd] invalid time format "%Y-%m-%dT%H:%M:%S.%LZ" for '2021-02-17T12:14:57.836555741Z'
[0] containers.var.log.containers.example-kk_default_count-ab9e855d67b387ca0d2cf624e7b90a0f593ff7fd0a0a5ab0c50afce12eb9524b.log: [1613564097.836678512, {"stream"=>"stdout", "logtag"=>"F", "log"=>"example-kk12: 5066 Wed Feb 17 12:14:57 UTC 2021"}]
[2021/02/17 12:14:58] [error] [parser] cannot parse '2021-02-17T12:14:58.837219052Z'
[2021/02/17 12:14:58] [ warn] [parser:containerd] invalid time format "%Y-%m-%dT%H:%M:%S.%LZ" for '2021-02-17T12:14:58.837219052Z'
```

**Comment by kkujawa-sumo** [comment]
Fixed by #1441
